operators/deidentification/pixelator/test_pixelator.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from holoscan.core import Operator, _Operator
```

Comment:
Not ideal to rely on a "private"-style underscore name outside of Holoscan SDK itself. In a future SDK release, we can possibly expose this as `OperatorBase` and have it be part of the public API to enable this use case.

We might consider updating to this so it would work both now and once that change is made upstream. We can also leave it as-is for now, but then we may forget to come back and update it later...

```py
from holoscan.core import Operator
try:
    from holoscan.core import OperatorBase
except ImportError:
    # use the private name was available in older versions of the SDK
    from holoscan.core import _Operator as OperatorBase
```

Reply:
Thanks, @grlee77! That's a great point. Alternatively, since all the python operators are supposed to be inherited from `Operator` (and not `_Operator`), what about using `Operator` instead?
```python
assert isinstance(op, Operator)
```

Reply:
I am fine with this simpler change. It will work for this operator and any other native Python ones.

The `_Operator` base class would only be needed to check the type of Python operators that are just pybind11 bindings on top of a C++ operator (i.e. like `FormatConverterOp`, `HolovizOp`, etc. in the core SDK).